template.env

```diff
@@ -0,0 +1,7 @@
+USER=
+PASSWORD=
+HOST=coaracp.austintexas.gov
+PORT=1521
+SERVICE=racweb_taf.world
+PGREST_ENDPOINT=https://atd-postgrest.austinmobility.io/legacy-scripts/traffic_reports
```

Comment:
i think we should probably pull the DB deets out of this file (and that github comment :/)

Reply:
okay, removed!